Do not use epmd out of the box to prevent killing system's epmd

Index: ejabberdctl.cfg.example
--- ejabberdctl.cfg.example.orig
+++ ejabberdctl.cfg.example
@@ -50,7 +50,7 @@
 #
 # Default: not defined
 #
-#ERL_DIST_PORT=5210
+ERL_DIST_PORT=5210
 
 #.
 #' FIREWALL_WINDOW: Range of allowed ports to pass through a firewall
